### PR TITLE
Security Sprite Update

### PR DESCRIPTION
**Pull request by sleepyyapril**
# Description

Contains:
- https://github.com/DeltaV-Station/Delta-v/pull/1901
- https://github.com/DeltaV-Station/Delta-v/pull/1753
- https://github.com/DeltaV-Station/Delta-v/pull/160

---

# Changelog

:cl:
- add: Added new security sprites and clothes!

### CODE REVIEW

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml [hunk 1]
```diff
@@ -17,8 +17,11 @@
     ClothingUniformJumpsuitSecBlue: 3
     ClothingUniformJumpskirtSecBlue: 3 # DeltaV - new skirt sprited, added to secdrobe
     ClothingHeadsetSecurity: 3
-    ClothingOuterWinterSec: 2
-    ClothingOuterCoatGreatcoat: 2 # DeltaV - added greatcoats to SecDrobe. Surplus, reminds the officers of the good times.
+    ClothingHeadsetPrison: 3 # DeltaV - prison headsets in secdrobe
+    ClothingOuterCoatStasec: 2 # DeltaV - replace ClothingOuterStasecSweater # DeltaV - add sweaters to secdrobe
+    ClothingOuterStasecStasec: 2 # DeltaV - replace ClothingOuterWinterSec
+    ClothingOuterCoatStasec: 2 # DeltaV - replace ClothingOuterWinterSec
+    ClothingOuterGreatcoatStasec: 2 # DeltaV - added greatcoats to SecDrobe. Surplus, reminds the officers of the good times.
```
Comment (TadJohnson00) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml:24]: All of this needs to get cleaned up. I suggest using the following:
```diff
...
    ClothingHeadsetPrison: 3 # DeltaV - prison headsets in secdrobe
+    ClothingOuterStasecSweater: 2 # DeltaV - add sweaters to secdrobe
+    ClothingOuterCoatStasec: 2 # DeltaV - replace ClothingOuterWinterSec
+    ClothingOuterGreatcoatStasec: 2 # DeltaV - added greatcoats to SecDrobe. Surplus, reminds the officers of the good times.
##    ClothingOuterArmorBasic: 2 # DeltaV - moved body armour from SecDrobe to SecTech
...
 ```

**Review by TadJohnson00 — COMMENTED** [review]
Fixing this should probably fix the errors you're running into. No promises.

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml (hunk 1 above)
Comment (sleepyyapril) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/secdrobe.yml:24]: Yeah, I was a bit worn out from jumping EE 10 engine versions pffft. Thanks!

**Comment by sleepyyapril** [comment]
bad merge...

**Comment by sleepyyapril** [comment]
i fix tmrw

**Comment by sleepyyapril** [comment]
> YAML linter doesn't seem to like the deletions in Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml :winks:

i merged bad, i'll fix it by merging good!

**Pull request closed without merging**